tools/train.py

```diff
@@ -129,7 +129,7 @@ def main():
                           '"auto_scale_lr.enable" or '
                           '"auto_scale_lr.base_batch_size" in your'
                           ' configuration file. Please update all the '
-                          'configuration files to mmdet >= 2.24.1.')
```

Comment:
we can keep this to be 2.24.1 since in 2.24.1 people can find auto_scale_lr config